server/routes/connections.js

```diff
@@ -2,11 +2,13 @@ import _ from 'lodash';
 import { Router } from 'express';
 
 import multipartRequest from '../lib/multipartRequest';
+// only fetch one page of connections
+const CONNECTIONS_FETCH_LIMIT = 50;
 
 export default (scriptManager) => {
   const api = Router();
   api.get('/', (req, res, next) => {
-    multipartRequest(req.auth0, 'connections', { strategy: 'auth0', fields: 'id,name,strategy,options' })
+    multipartRequest(req.auth0, 'connections', { strategy: 'auth0', fields: 'id,name,strategy,options' }, { perPage: 100, limit: CONNECTIONS_FETCH_LIMIT})
```

Comment:
do we need `perPage: 100`? Should not the default 50 be fine?

Reply:
Yeah I agree with this, we only use 20 anyway, and we'd still only make one request whether it was 50 or 100. I'll remove this and we can use the default.

Reply:
[b623d9f](https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/292/commits/b623d9f3a4f49df44e1af1f57c08748891c435ff)